lib/ddtrace/workers/runtime_metrics.rb

```diff
@@ -0,0 +1,58 @@
+require 'forwardable'
+
+require 'ddtrace/environment'
+require 'ddtrace/runtime/metrics'
+
+require 'ddtrace/worker'
+require 'ddtrace/workers/polling'
+
+module Datadog
+  module Workers
+    # Emits runtime metrics asynchronously on a timed loop
+    class RuntimeMetrics < Worker
+      extend Forwardable
+      include Workers::Polling
+
+      attr_reader \
+        :metrics
+
+      def initialize(options = {})
+        @metrics = options.fetch(:metrics, Runtime::Metrics.new)
+
+        # Workers::Async::Thread settings
+        self.fork_policy = options.fetch(:fork_policy, Workers::Async::Thread::FORK_POLICY_STOP)
```

Comment:
We don't want forks to collect runtime metrics by default; this policy prevents the runtime metrics worker from restarting and collecting metrics.